include/revng/PipelineC/Prototypes.h

```diff
@@ -37,6 +37,12 @@ bool rp_initialize(int argc,
                    int libraries_count,
                    const char *libraries_path[]);
 
+/**
+ * Should be called on clean exit to clean up all LLVM-related stuff used by
+ * revng-pipeline.
+ */
+bool rp_shutdown();
```

Comment:
Make sure the return values of `rp_shutdown` and `rp_initialize` are documented.

Reply:
Done